src/Raven.Server/Documents/PeriodicBackup/Restore/RestoreBackupTask.cs

```diff
@@ -90,6 +90,9 @@ public async Task<IOperationResult> Execute(Action<IOperationProgress> onProgres
                         onProgress,
                         result);
 
+                    if (restoreSettings != null && _restoreConfiguration.SkipIndexesImport)
```

Comment:
Shouldn't the logic here be reversed?

Reply:
Okay, the code is correct, but _really_ confusing. 
Why do you need it here if this is empty all the time?

Reply:
done